source/shared/version.h

```diff
@@ -1,58 +1,58 @@
-#ifndef VERSION_H
-#define VERSION_H
-//---------------------------------------------------------------------------------------------------------------------------------
-// File: version.h
-// Contents: Version number constants
-//
-// Microsoft Drivers 4.3 for PHP for SQL Server
-// Copyright(c) Microsoft Corporation
-// All rights reserved.
-// MIT License
-// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files(the ""Software""), 
-//  to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, 
-//  and / or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions :
-// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
-// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, 
-//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER 
-//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS 
-//  IN THE SOFTWARE.
-//---------------------------------------------------------------------------------------------------------------------------------
-
-// helper macros to stringify the a macro value
-#define STRINGIFY(a) TOSTRING(a)
-#define TOSTRING(a) #a
-
-
-// Increase Major number with backward incompatible breaking changes.
-// Increase Minor with backward compatible new functionalities and API changes.
-// Increase Patch for backward compatible fixes.
-#define SQLVERSION_MAJOR 5
-#define SQLVERSION_MINOR 0
-#define SQLVERSION_PATCH 0
-#define SQLVERSION_BUILD 0
-
-// Semantic versioning pre-release 
-// for stable releases should be empty
-// "-RC" for release candidates
-// "-preview" for ETP 
-#define SEMVER_PRERELEASE

```

Comment:
"-preview"should be set as value of SEMVER_PRERELEASE, and should be tested using sql_clinet_info()